### ISSUE TITLE
Accordion retracts when an input element is clicked inside of it

### ISSUE THREAD

**Issue by oliveiraddb** [appnexus/lucid#401]
Given this example:

```
<Accordion>
        <Accordion.Item>
          <Accordion.Header>Peter Venkman</Accordion.Header>
          <TextField/>
        </Accordion.Item>
        <Accordion.Item>
          <Accordion.Header>Peter Venkman</Accordion.Header>
          <input/>
        </Accordion.Item>
<Accordion>
```

When i click on the input element inside of an Accordion Item, it retracts. My guess is that this happens because accordion has an `onSelect` event that conflicts with some property of the inputs.

**Comment by sodiumjoe** [comment]
i bet this is the same issue as #398 , `Accordion` also doesn't use `omitProps` on `...passThroughs`

**Comment by sodiumjoe** [comment]
confirmed, same issue, fixed in #403
